QNNAKWUE/taskListApp
# Creates /api/v1/auth/signup

## What does this PR do?
- Creates `POST: /api/v1/auth/signup/`

## Description of Task taken to complete??
- setup directories
- implement the feature with a fake database


## How should this be manually Tested?
- Clone this repo
- `git checkout ft-user-should-signup`
- `Run yarn install`
- start the server `yarn run start`
- Open postman on port 4200 , then POST localhost:6010 /api/v1/auth/signup


### Any Background Context you want to provide?
the request body must contain the following entities:
> {
"fullname" : String
"email": String,
"password" : String,
"confirmPassword": String,

> } 

- You must setup all .env configs in the .env.example file

- To see test cases Run `yarn test` in the command line
